<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddComputerDialog</class>
 <widget class="QDialog" name="AddComputerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <widget class="QLineEdit" name="nameBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Build year</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="typeBox">
         <item>
          <property name="text">
           <string>Mechanic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Electronic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Transistor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Quantum</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QSpinBox" name="buildYearBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QCheckBox" name="computerBuiltBox">
         <property name="text">
          <string>Computer built</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
